invenio_assets/assets/build/webpack.config.js

```diff
@@ -213,11 +206,11 @@ var webpackConfig = {
       "window.jQuery": "jquery/src/jquery"
     }),
     // Write manifest file which Python will read.
-    new ManifestPlugin({
-      fileName: "manifest.json",
-      stripSrc: true,
-      publicPath: config.build.assetsURL
-    })
+    new BundleTracker({
```

Comment:
Is this potentially a big change?


Would people e.g. need to change this:

``{{ webpack['theme'] }}``

to this:

``{{ webpack['theme.css'] }}``

**Chunking**
What impact does it have on existing chunking?

This is just to ensure that we actually properly document the upgrade.

Reply:
In the manifest loader, each entry's chunk paths [are split into `.js` and `.css`](https://github.com/inveniosoftware/pywebpack/blob/b78bc745cd0d9c081da1b7356d254d0089746ed9/pywebpack/manifests.py#L193-L205), so usage remains the same. What changes is that there is no need to actually include the runtime and vendor chunks explicitly, they are automatically picked up as dependencies of each entry:

```diff
- {{ webpack['manifest'] }}
- {{ webpack['vendor'] }}
{# This will include the above #}
{{ webpack['base'] }}
```